.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,45 +1,29 @@
-<!--- Provide a general summary of your changes in the Title above -->
+<!--- Title: Provide a general summary of your changes in the Title above -->
+<!--- Reviewers: Include @karstens, @GarthDB, @mrcjhicks, @lynnhao -->
 
 ## Description
 
-<!--- Describe your changes in detail -->
+<!--- Describe your changes in detail, including a list of changes -->
 
 ## Related Issue
 
 <!--- This project only accepts pull requests related to open issues -->
-<!--- If suggesting a new feature or change, please discuss it in an issue first -->
+<!--- If suggesting a new feature or change, please discuss it in #spectrum_tokens_talk or design workshop, first -->
 <!--- If fixing a bug, there should be an issue describing it with steps to reproduce -->
-<!--- Please link to the issue here: -->
-
-## Motivation and Context
-
-<!--- Why is this change required? What problem does it solve? -->
-
-## How Has This Been Tested?
-
-<!--- Please describe in detail how you tested your changes. -->
-<!--- Include details of your testing environment, and the tests you ran to -->
-<!--- see how your change affects other areas of the code, etc. -->
-
-## Screenshots (if appropriate):
+<!--- Please link to the issue on the next line: -->
 
 ## Types of changes
 
 <!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
 
-- [ ] Bug fix (non-breaking change which fixes an issue)
-- [ ] New feature (non-breaking change which adds functionality)
-- [ ] Breaking change (fix or feature that would cause existing functionality to change)
+- [ ] Bug fix (patch; non-breaking change which fixes an issue) <!--- Example: fixing incorrect metadata -->
+- [ ] New feature (minor; non-breaking change which adds functionality) <!--- Examples: editing a token value, adding a new token -->
```

Comment:
what i'm not 100% sure about so would love to clarify is whether "editing a token value" is always a minor, for example, if it's going to cause layout shifts, should it be considered a breaking change? or is that separate from this versioning (which might be specific to only our tokens data, not overall design version)??

Reply:
I think we should think about it in the context of SemVer. So 1.2.3:
- 1 = major: adding a new design token. 
- 2 = minor: bug fix or small enhancement, editing an existing design token. **_this is a little tricky when it comes to visual changes like spacing tokens, color, typography._** 
- 3 = patch: small bug fix or typo correction, fixing incorrect metadata of an existing design token. 

I think in either of the versioning levels, it could trigger a change in the S/Color or S2/Color Figma libraries. It shouldn't be a problem so as long as when we inherit updates from those libraries we include a sentence in the changelogs of S/Desktop, S/Mobile and S2/Desktop.

Reply:
This is how we describe the semver for Spectrum Tokens. I think it could work here too: https://github.com/adobe/spectrum-tokens/tree/main/packages/tokens#versioning